admin-center/activate/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: Activate
+sidenav: admin_center
+---
+* [Code snippets]({{ site.baseurl }}/manual/code.html)
+* [Search Results API instructions and access key](https://open.gsa.gov/api/ searchgov-results/)\*
+* [Type-ahead API instructions](https://open.gsa.gov/api/searchgov-suggestions/)\*
+* [i14y API instructions]({{ site.baseurl }}/developer/i14y.html)\*
+
+\* These APIs are available for use on official government websites only. You must be a Search.gov customer. [Sign in](https://search.usa.gov/sites) is required. Keys and instructions for each can be found under [Admin Center](https://search.usa.gov/sites) > YourSite > Activate Search.
+
+{% include under-construction.html %}
```

Comment:
why doesn't this page show my changes? hope they didn't get lost.

Reply:
Going back through the commits I don't think your changes were ever committed, they might be lost :/